docs/src/main/paradox/server-side/http2.md

```diff
@@ -39,7 +40,20 @@ Java
 
 Note that `bindAndHandle` currently does not support HTTP/2, you must use `bindAndHandleAsync`.
 
-## Testing with cURL
+### HTTP/2 without HTTPS
+
+While un-encrypted connections are allowed by HTTP/2, clients that support this are rare.
```

Comment:
not sure about this claim; rather that it is not recommended to be used for real outside of testing?